src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/oops/ConstantPool.java

```diff
@@ -464,16 +467,35 @@ public int getMethodTypeIndexAt(int i) {
     return res;
   }
 
-  /** Lookup for multi-operand (InvokeDynamic, Dynamic) entries. */
-  public short[] getBootstrapSpecifierAt(int i) {
+  public int getBootstrapMethodsCount() {
+    U2Array operands = getOperands();
+    int count = 0;
+    if (operands != null) {
+      count = getOperandOffsetAt(operands, 0) / 2;
+    }
```

Comment:
Nit: Could you, please, add a small comment why the bootstrap methods count is calculated this way?

Reply:
Done.